cli/tests/node_compat/test/parallel/test-http-request-create-url-from-options.js

```diff
@@ -0,0 +1,32 @@
+// deno-fmt-ignore-file
+// deno-lint-ignore-file
+
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const http = require('http');
+const { urlToHttpOptions } = require('url');
+
+const testPath = '/foo?bar';
+
+const server = http.createServer(common.mustCall((req, res) => {
+  assert.strictEqual(req.url, testPath);
```

Comment:
It seems like you only assert path path of URL but the fix is for hostname part, is it intentional?

Reply:
The assert added here seems useless, in fact, I just need to ensure that `http.request` is working properly